deps.edn

```diff
@@ -0,0 +1,21 @@
+{:deps {org.clojure/clojure {:mvn/version "1.11.1"}
+        org.clojure/math.numeric-tower {:mvn/version "0.0.5"}
+        clj-time/clj-time {:mvn/version "0.15.2"}
+        com.andrewmcveigh/cljs-time {:mvn/version "0.5.2"}}
+ :paths ["src" "test"]
```

Comment:
If `"test"` is included here, it will be shipped in the JAR/gitlib to everyone.

```suggestion
 :paths ["src"]
```

Reply:
Thank you for spotting this. Fixed.